CMakeLists.txt

```diff
@@ -89,15 +89,19 @@ bob_public_dep(pybind11)
 
 if(Omega_h_USE_Kokkos)
   if ("CUDA" IN_LIST Kokkos_DEVICES)
+      add_definitions(-DOmega_h_USE_CUDA)
```

Comment:
Since we include `Omega_h_config.h`, which CMake creates (via `bob.cmake`, IIRC), in `Omega_h_kokkos.hpp`, we don't need to add these `add_definitions(...)` calls.